marklogic-data-hub-central/ui/src/components/common/hc-table/hc-table.tsx

```diff
@@ -6,6 +6,7 @@ import {DropdownButton, Dropdown} from "react-bootstrap";
 import styles from "./hc-table.module.scss";
 import "./hc-table.scss";
 import {CaretDownFill, CaretUpFill, ChevronDown, ChevronRight} from "react-bootstrap-icons";
+import {HCButton} from "..";
 
 interface Props {

```

Comment:
What do you think about typing the columns prop? It seems to have the same interface in almost all implementations, right?

Reply:
Good catch! There are a couple of props (`columns`, `nestedParams`, among a few others) that we could benefit from typing them explicitly, even for self-documenting purposes and it would be a good fit. Personally, I was worried that starting off that task just a few table replacements already merged would require us to conduct some refactors over with every table PR. We might approach this work as technical debt ticket if it works for you.